pallets/security/Cargo.toml

```diff
@@ -14,14 +14,13 @@ sha2 = {version = "0.8.2", default-features = false}
 frame-support = {git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false}
 frame-system = {git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false}
 sp-core = {git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false}
-sp-runtime = {git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false}
 sp-std = {git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false}
 
 
 
 [dev-dependencies]
 mocktopus = "0.8.0"
-sp-io = {git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false}
```

Comment:
we need the `std` feature, to avoid externalities issues:
```
error: cannot find macro `thread_local` in this scope
  --> /home/runner/.cargo/git/checkouts/polkadot-sdk-67b6d29d759a1af6/e51a91f/substrate/primitives/externalities/src/scope_limited.rs:22:1
   |
22 | environmental::environmental!(ext: trait Externalities);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `thread_local` is in scope, but it is an attribute: `#[thread_local]`
   = note: this error originates in the macro `$crate::thread_local_impl` which comes from the expansion of the macro `environmental::environmental` (in Nightly builds, run with -Z macro-backtrace for more info)
```

See [failed CI](https://github.com/pendulum-chain/spacewalk/actions/runs/10934934905/job/30356549516#step:6:1005)